### ISSUE TITLE
zappa tail <environment> failing

### ISSUE THREAD

**Issue by fifthecho** [Miserlou/Zappa#553]
In Zappa 0.32.0, `zappa tail stage` would pull logs for the `stage` environment of an application.

However, with Zappa 0.33.0, I am getting:

```
zappa tail stage
usage: zappa [-h] [-v] [-a APP_FUNCTION] [-s SETTINGS_FILE]
             {certify,deploy,init,invoke,manage,rollback,schedule,status,tail,undeploy,unschedule,update}
             ...
zappa: error: unrecognized arguments: stage
```

**Comment by fifthecho** [comment]
It's worth mentioning that this only appears to be affecting the `tail` subcommand: updates, status, certify, etc. all seem to be working, but tail is not.

**Comment by Miserlou** [comment]
Fixed in 0.32.1, publishing now..

**Comment by Amrit0991** [comment]
I'm getting this same issue with `zappa unschedule` as well. @Miserlou would you like me to submit a PR?
